x-pack/plugins/security_solution/docs/prebuilt_rules_customization_rfc.md

```diff
@@ -0,0 +1,2867 @@
+  },
+  {
+    ecs: false,
+    name: "host.user.id",
+    type: "keyword"
+  },
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td rowspan=2 style="border-right:3px solid black">Keep user changes if element conflicts and add new matching element <br><br>
+      <ul><li><b>1st elem: (A B C - keep current) </li><li>2nd elem: (_ A A - keep current/target)</b></li></ul>
+      </td>
+      <td>
+      <pre>
+[
+  logs-*,
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  cluster_one:logs-*,
+  new-target
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  logs-new-*,
+  new-target
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  cluster_one:logs-*,
+  new-target
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "unknown"
+  }
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  },
+  {
+    ecs: false,
+    name: "my.new.field",
+    type: "keyword"
+  }
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "keyword"
+  },
+  {
+    ecs: false,
+    name: "my.new.field",
+    type: "keyword"
+  },
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  },
+  {
+    ecs: false,
+    name: "my.new.field",
+    type: "keyword"
+  } ,
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td rowspan=2 style="border-right:3px solid black">Keep user changes if element conflicts and add both new conflicting elements <br><br>
+      <ul><li><b>1st elem: (A B C - keep current) </li><li>2nd elem: (_ A B - keep both current and target)</b></li></ul>
+      </td>
+      <td>
+      <pre>
+[
+  logs-*,
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  cluster_one:logs-*,
+  new-target
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  logs-new-*,
+  elastic-index
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  cluster_one:logs-*,
+  new-target,
+  elastic-index
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "unknown"
+  }
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  },
+  {
+    ecs: false,
+    name: "my.new.field",
+    type: "keyword"
+  },
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "keyword"
+  },
+  {
+    ecs: true,
+    name: "new.elastic.ip",
+    type: "ip"
+  }
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
```

Comment:
This example looks "broken". Names `host.user.id` and `process.executable` aren't presented in the columns to the left.